src/nxrunner/src/nxrunner.cpp

```diff
@@ -481,8 +481,11 @@ std::vector<std::string> GetPythonPluginList()
   {
     return {};
   }
-
-  return StringUtilities::split(var, ';');
+#if defined(Q_OS_WIN)
```

Comment:
I think I have a solution that will let just deal with one environment variable instead of both `PYTHONPATH` and `SIMPLNX_PYTHON_PLUGINS` for this that I'm testing on my reload branch.

Reply:
OK. We can leave this in for now and then change over when you get your branch in.